tensorflow/docs_src/deploy/s3.md

```diff
@@ -0,0 +1,41 @@
+# How to run TensorFlow on S3
+
+This document describes how to run TensorFlow on S3 file system.
+
+## S3
+
+We assume that you are familiar with @{$reading_data$reading data}.
+
+To use S3 with TensorFlow, change the file paths you use to read and write
+data to an S3 path. For example:
+
+```python
+filename_queue = tf.train.string_input_producer([
```

Comment:
Can you use a tf.data example instead? We'll deprecate the old style inputs.

Reply:
Thanks @jhseu, done.